Cargo.toml

```diff
@@ -49,9 +49,9 @@ nix = "0.20.0"
 openat = "0.1.19"
 openat-ext = "^0.1.13"
 os-release = "0.1.0"
-ostree = { version = "0.9.1", features = ["v2020_4"] }
+ostree = { version = "0.10.0", features = ["v2021_1"] }
 ostree-sys = "0.7.2"
-ostree-host = { path = "rust/ostree-host" }
+ostree-ext = { git = "https://github.com/cgwalters/ostree-ext", rev = "1d69adde3825d13e100734bc82883ac4bf3f5be9" }
```

Comment:
Should that URL be updated now?

Reply:
Oh yep definitely, I meant to mark this as draft to start.  I mostly wanted to see the CI results.

Will work on getting things published to crates.io soon.